openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EventSubscriptionRepository.java

```diff
@@ -0,0 +1,201 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.openmetadata.service.jdbi3;
+
+import com.lmax.disruptor.BatchEventProcessor;
+import java.io.IOException;
+import java.util.Comparator;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.openmetadata.schema.entity.events.EventFilterRule;
+import org.openmetadata.schema.entity.events.EventSubscription;
+import org.openmetadata.schema.entity.events.SubscriptionStatus;
+import org.openmetadata.schema.type.EntityReference;
+import org.openmetadata.service.Entity;
+import org.openmetadata.service.events.EventPubSub;
+import org.openmetadata.service.events.subscription.AlertUtil;
+import org.openmetadata.service.events.subscription.SubscriptionPublisher;
+import org.openmetadata.service.resources.events.subscription.EventSubscriptionResource;
+import org.openmetadata.service.util.EntityUtil.Fields;
+
+@Slf4j
+public class EventSubscriptionRepository extends EntityRepository<EventSubscription> {
```

Comment:
Everywhere let's use one of these names - either EventSubscription or just Subscription.